app/helpers/check_answers_helper.rb

```diff
@@ -11,6 +11,18 @@ def display_answered_questions_summary(subsection, case_log, current_user)
     end
   end
 
+  def can_change_scheme_answer?(attribute_name, scheme)
+    editable_attributes = current_user.support? ? ["Name", "Confidential information", "Housing stock owned by"] : ["Name", "Confidential information"]
+    !scheme.confirmed? || editable_attributes.include?(attribute_name) 
+  end
+
+  def get_location_change_link_href(scheme, location)
+    if location.confirmed?
+      "/schemes/#{scheme.id}/locations/#{location.id}/edit-name"
```

Comment:
```suggestion
      location_edit_name_path(scheme, location)
```

Can we use route helper methods for both conditions?